superset-frontend/plugins/plugin-chart-echarts/src/Heatmap/controlPanel.tsx

```diff
@@ -224,6 +225,31 @@ const config: ControlPanelConfig = {
         ['y_axis_format'],
         ['x_axis_time_format'],
         ['currency_format'],
+        [<ControlSubSectionHeader>Border</ControlSubSectionHeader>],
+        [
+          {
+            name: 'element_border',
+            config: {
+              type: 'ColorPickerControl',
+              label: t('Border Color'),
+              renderTrigger: true,
+              description: t('Select border color for Heatmap elements'),
+              default: { r: 255, g: 255, b: 255, a: 1 },
+            },
+          },
+        ],
+        [
+          {
+            name: 'enable_border',
```

Comment:
If the border color can be transparent by default, you don't need a checkbox to enable the border, you just need to set a color. You could also have a control for border width with a default value of 0 indicating no border.